docs/overview/use_cases.md

```diff
@@ -86,8 +86,9 @@ blocking barriers and then a more loosely coupled execution.
 <b>Blocking Barrier</b>
 
 When the simulation job starts, one task opens the checkpoint container
-and fetches the current global HCE. It thens obtains an epoch hold and
-shares the data (the container handle, the current LHE and global HCE)
+and fetches the current global HCE (highest committed epoch).
+It thens obtains an epoch hold and shares the data (the container handle,
```

Comment:
s/thens/then ?

Reply:
Thanks for catching this.  I will fix it.
I also have a few other minor typographical / clarification changes.  Is it okay to include them in the PR as a general documentation cleanup, or should I open a separate PR for these (and maybe subsequent) changes?

Reply:
you can use this PR, or open a new one, whatever makes most sense from a topic perspective.
it's generally better to push more smaller PRs, as that's easier to review.